ajax/libs/jquery.flipster/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "jquery.flipster",
+  "version": "1.1.2",
+  "homepage": "http://brokensquare.com/Code/jquery-flipster/demo/",
+  "description": "Responsive coverflow-style carousel plugin for jQuery",
+  "filename": "jquery.flipster.min.js",
+  "author": "Adrien Delessert <a@drien.com>",
+  "keywords": [
+    "coverflow",
+    "jquery",
+    "carousel",
+    "flipster",
+    "jquery-plugin",
+    "ecosystem:jquery"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com:drien/jquery-flipster.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/drien/jquery-flipster.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "+(js|css)/*.+(js|css|map)"
```

Comment:
Can you tell me what's the pattern for? Like which file in which version, as I didn't see it work in the latest version, and I assume it should at least fetched the non-minified js src there?